src/model/game/GameManager.ts

```diff
@@ -794,6 +794,12 @@ export default class GameManager {
             "https://thunderstore.io/c/gorebox/api/v1/package-listing-index/", EXCLUSIONS,
             [new StorePlatformMetadata(StorePlatform.STEAM, "2027330")], "GoreBox.png",
             GameSelectionDisplayMode.VISIBLE, GameInstanceType.GAME, PackageLoader.BEPINEX, [""]),
+        
+        new Game("TCG Card Shop Simulator", "TCG Card Shop Simulator", "TCG Card Shop Simulator",
+            "TCG Card Shop Simulator", ["Card Shop Simulator.exe"], "Card Shop Simulator_Data",
+            "thunderstore.io/c/tcg-card-shop-simulator/api/v1/package/", "https://raw.githubusercontent.com/ebkr/r2modmanPlus/master/modExclusions.md",
```

Comment:
- The `tsUrl` parameter is outdated (see above for an example)
- Use the `EXCLUSIONS` const for `exclusionUrl` parameter

Reply:
Will fix